public/app/features/profile/UserOrganizations.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react';
+import { User } from 'app/types';
+import { UserOrg } from 'app/core/utils/UserProvider';
+import { LoadingPlaceholder, Button } from '@grafana/ui';
+
+export interface Props {
+  user: User;
+  orgs: UserOrg[];
+  isLoading: boolean;
+  loadOrgs: () => void;
+  setUserOrg: (org: UserOrg) => void;
+}
+
+export class UserOrganizations extends PureComponent<Props> {
+  componentDidMount() {
+    this.props.loadOrgs();
+  }
+
+  render() {
+    const { isLoading, orgs, user } = this.props;
+
+    if (isLoading) {
+      return <LoadingPlaceholder text="Loading organizations..." />;
+    }
+
+    return (
+      <>
+        {orgs.length > 0 && (
+          <>
+            <h3 className="page-sub-heading">Organizations</h3>
```

Comment:
Instead of using css classes take a look at emotion used in for instance https://github.com/grafana/grafana/blob/master/public/app/features/explore/TypeaheadInfo.tsx